client/my-sites/people/people-list-item/index.jsx

```diff
@@ -42,13 +46,13 @@ export default React.createClass( {
 			this.userHasPromoteCapability() &&
 			! this.props.isSelectable
 		);
-	},
+	};
 
 	render() {
 		const canLinkToProfile = this.canLinkToProfile();
 		return (
 			<CompactCard
-				{ ...omit( this.props, 'className', 'user', 'site', 'isSelectable', 'onRemove' ) }
+				{ ...omit( this.props, 'className', 'user', 'site', 'isSelectable', 'onRemove', 'translate', 'moment', 'numberFormat' ) }
```

Comment:
at what point do we show _what should pass through_ instead of _what shouldn't_?

we have a bunch of these `omit()` examples in the codebase but it's confusing because it doesn't show anything about what actually gets passed down the prop chain